php/libraries/Utility.class.inc

```diff
@@ -117,9 +118,9 @@ class Utility
      *
      * @return array an associative array("center ID" => "site name")
      */
-    static function getSiteList($study_site = true)
+    static function getSiteList(bool $study_site = true): array
```

Comment:
bool|null in the doc block

Reply:
As discussed in the other PR, this param isn't actually nullable.